test/mqtt_store.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+var mqtt = require('../lib/connect')
+
+describe('store in mqtt', function () {
```

Comment:
Can you use a more describing name for this? It's not really `mqtt`, it's `connect()` from a Node perspective.

Reply:
How about this?

```javascript
describe('store in lib/connect/index.js (webpack entry point)', function () {
  it('should create store', function (done) {
    done(null, new mqtt.Store())
  })
})
```

Reply:
thanks!

Reply:
That will be perfect, can you push?